doc/api/crypto.md

```diff
@@ -1121,6 +1121,15 @@ exposes different functions.
 Most applications should consider using the new `KeyObject` API instead of
 passing keys as strings or `Buffer`s due to improved security features.
 
+### keyObject.asymmetricKeySize
+<!-- YAML
+added: REPLACEME
+-->
+* {number}
+
+For secret keys, this property represents the size of the key in bytes. This
```

Comment:
Copy/paste error? It seems to be at odds with the implementation and the tests. Do you mean 'For public and private keys'?

Reply:
not directly. i mixed up the names of the classes. i aligned it with `keyObject.asymmetricKeyType` where we use already "For asymmetric keys". Maybe we should rename "For secret keys, " in `keyObject.symmetricKeySize` to "For symmetric keys" to make i more clear? I'd create an addional PR for that if we agree on it.